src/sparseml/core/recipe/container.py

```diff
@@ -53,7 +54,9 @@ class RecipeContainer:
 
     def update(
```

Comment:
Needed to update this function to accept a List of Modifiier(s); my personal addition is maybe 5-10 lines, the entire function is shown as a change because I de-indented almost the entire function by exiting early